db/schema.rb

```diff
@@ -120,6 +120,7 @@
     t.string   "encrypted_otp_secret_key_salt", limit: 255
     t.string   "direct_otp"
     t.datetime "direct_otp_sent_at"
+    t.boolean  "sms_otp_delivery",                         default: true
```

Comment:
Totally agree re: column naming, however I think persistence of preference is yet to be determined. I am removing it from this PR.